rpc/server.go

```diff
@@ -88,7 +90,13 @@ func (s *Server) Send(ctx context.Context, in *proto.NotificationRequest) (*prot
 	if in.Data != nil {
 		notification.Data = map[string]interface{}{}
 		for k, v := range in.Data.Fields {
-			notification.Data[k] = v
+			if x, ok := v.GetKind().(*structpb.Value_StringValue); ok {
+				notification.Data[k] = x.StringValue
+			} else if x, ok := v.GetKind().(*structpb.Value_NumberValue); ok {
+				notification.Data[k] = x.NumberValue
+			} else if x, ok := v.GetKind().(*structpb.Value_BoolValue); ok {
+				notification.Data[k] = x.BoolValue
+			}
```

Comment:
what about other types:list, null, struct? may be add some logging for else cases?

Reply:
That's why the PR is on hold. I am no expert in Go, which is why I need help from you.
NullValue and ListValue might be easy. However, structs seem to be difficult for me.

Reply:
Hi, also not a go expert myself, but maybe [protojson](google.golang.org/protobuf/encoding/protojson) could be used here so simplify things. That would mean notification.Data is a []byte field instead map[string]interface{} and would be filled by calling notification.Data, err = protojson.Marshal(in.Data).